src/features/zaaksysteem/service.ts

```diff
@@ -82,8 +83,25 @@ export function useZaaksysteemService() {
     return { withoutFetcher, withFetcher };
   };
 
+  const getZaak = (id: string): ServiceData<ZaakDetails> => {
```

Comment:
This function doesn't actually return a zaak, it's only returning the `id`.

Reply:
hij retourneert een zaakdetail object. daar zit nog bijna niets in, maar vast vooruitlopend op de volgende stories wordt er alvast een object geretourneerd.